source/django_package/domain/import_datas/intermediate_layer.py

```diff
@@ -0,0 +1,18 @@
+from source.django_package.domain.import_datas import parsing_config_file
+from source.django_package.domain.import_datas.import_csv import import_csv_data
+
+
+class Adapter:
+    data_type = parsing_config_file.ReadConfigFile.data_type[0]
```

Comment:
Putting stuff that is run-time generated in the class property is very, very risky and generally wrong. Class property is defined when the class is loaded into memory (when the app starts) and will not change later on (contrary to instance properties which are defined in `__init__` method). It is difficult to predict in which circumstances this would fail, but it definitely might.